packages/app-page-builder-elements/src/renderers/form/types.ts

```diff
@@ -77,8 +84,12 @@ export interface ErrorResponse {
 export type FormLayoutComponentProps<T = any> = {
     getFieldById: Function;
     getFieldByFieldId: Function;
-    getFields: () => FormRenderComponentDataField[][];
+    getFields: (stepIndex: number) => FormRenderComponentDataField[][];
     getDefaultValues: () => { [key: string]: any };
+    goToPreviousStep: () => void;
```

Comment:
I don't see `goToNextStep` here... hmmm 🤔 

It's basically because we don't need it? Because the switching to next step will be done by the `onSubmit` call?

Reply:
@adrians5j yes, because of that.

Reply:
I have revetred this change, and now I am passing this prop down to `DefaultFormLayout` so we can use it inside `submitForm` function.